eng/Version.Details.xml

```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
-  <ProductDependencies>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="3.1.0-rtm.19565.2" Pinned="true">
-      <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>65f04fb6db7a5e198d05dbebd5c4ad21eb018f89</Sha>
-    </Dependency>
-  </ProductDependencies>
   <ToolsetDependencies>
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.23110.3">
```

Comment:
This should be changed to match the entry in `global.json`. I recommend adding a `darc` subscription from Arcade's 6.0 branch, then manually triggering it to see what it updates. Ping me offline if you need help with that

Reply:
Thanks, yeah I want to talk to you about which channels this and main should participate in.

Reply:
Updated. Here are the current subscriptions.

Branch | Channel | Description
--- | --- | ---
main | .NET Eng - Latest | A hypothetical future version
release/6.5 | .NET 6 Eng | The version we're trying to ship
release/6.4 | .NET 3 Eng | Servicing branch for 6.4 (shipped with .NET 3.1)